packages/auth/src/grant/routes.ts

```diff
@@ -203,9 +222,19 @@ async function getGrantDetails(
   ctx.body = { access: grant.access }
 }
 
+interface StartQuery {
+  clientName: string
+  clientUri: string
+}
```

Comment:
`clientName` and `clientUri` should be added to the OpenAPI spec to be enforced:
https://github.com/interledger/rafiki/blob/18d2537a59ae9adb74d61b850132957b40f7caa6/packages/auth/src/openapi/id-provider.yaml#L41